services/ui-src/src/components/fields/ChoiceListField.test.tsx

```diff
@@ -67,47 +67,11 @@ const ChoiceListFieldWithNestedChildren = (
       label="Radio example"
       name="radio_choices"
       type="radio"
-      onChangeHandler={jest.fn()}
+      onChangeHandler={() => jest.fn()}
     />
   </div>
 );
 
-describe("Test ChoiceList component", () => {
-  test("ChoiceList renders as Checkbox", () => {
-    render(ChoiceListFieldCheckboxComponent);
-    expect(screen.getByText("Choice 1")).toBeVisible();
-    expect(screen.getByTestId("test-checkbox-list")).toBeVisible();
-  });
-
-  test("ChoiceList renders as Radio", () => {
-    render(ChoiceListFieldRadioComponent);
-    expect(screen.getByText("Choice 1")).toBeVisible();
-    expect(screen.getByTestId("test-radio-list")).toBeVisible();
-  });
-
-  test("ChoiceList allows checking checkbox choices", async () => {
-    const wrapper = render(ChoiceListFieldCheckboxComponent);
-    const checkboxContainers = wrapper.container.querySelectorAll(
-      ".ds-c-choice-wrapper"
-    );
-    const firstCheckbox = checkboxContainers[0].children[0] as HTMLInputElement;
-    expect(firstCheckbox.checked).toBe(false);
-    await userEvent.click(firstCheckbox);
-    expect(firstCheckbox.checked).toBe(true);
-  });
-
-  test("ChoiceList allows checking radio choices", async () => {
-    const wrapper = render(ChoiceListFieldRadioComponent);
-    const radioContainers = wrapper.container.querySelectorAll(
-      ".ds-c-choice-wrapper"
-    );
-    const firstRadio = radioContainers[0].children[0] as HTMLInputElement;
-    expect(firstRadio.checked).toBe(false);
-    await userEvent.click(firstRadio);
-    expect(firstRadio.checked).toBe(true);
-  });
-});
-
```

Comment:
These tests are already covered in RadioField and CheckboxField so no need to duplicate here